# python_exception
Exceptions in Python
